uiplib/utils.py

```diff
@@ -17,6 +19,20 @@ def make_dir(dirpath):
     if sys.platform.startswith('linux'):
         os.chmod(dirpath, 0o777)
 
+
+def exit_UIP(service=False):
+    pid_file = os.path.join(HOME_DIR, 'daemon-uip.pid')
+    if service:
```

Comment:
@nemaniarjun if there's no pid file then there's no service running right? and you check for a pid file :D